core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -1,8 +1,5 @@
 package org.testcontainers.containers;
```

Comment:
Regarding imports, there are some changes in ordering, but AFAICT these only match the IntelliJ defaults (with our `.editorconfig` tweaks layered on top for star imports).

TBH I'd prefer to go with the changed (hopefully stable) import order in this PR, rather than continue to constantly have the burden of maintaining the order (I don't know why the current order is as it is).

I think if we find ourselves continuing to run up against this then we should introduce a code formatter to our build (e.g. spotless gradle plugin + google formatter with AOSP style (4 space indent)).